rust/sealevel/libraries/access-control/Cargo.toml

```diff
@@ -7,11 +9,13 @@ edition = "2021"
 no-entrypoint = []
 
 [dependencies]
-solana-program = "=1.14.13"
+solana-program.workspace = true
+
+# Required to allow dependencies `getrandom` but to preserve determinism required by programs, see
+# https://github.com/solana-labs/solana/blob/master/docs/src/developing/on-chain-programs/developing-rust.md#depending-on-rand
+getrandom = { version = "0.2", features = ["custom"] }
```

Comment:
I added these possibly overgenerously - I know some of these were necessary at the time, but it's possible I added them to Cargo.tomls that didn't require them and/or it's no longer necessary after some of our changes

Could you make sure these are necessary in each Cargo.toml that it's been added to, and also possibly specify it as a workspace dependency instead of having `version = "0.2"` everywhere?

Reply:
much too generously indeed, it's not needed at all 😛